woocommerce/woocommerce
# Add link to documentation to Downloads section

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds a link to the Downloads documentation in the section's tooltip. 

**Design**
AkVNGImLgSqCObTQ3idVn7-fi-3301_392136

![Screenshot 2023-12-01 at 14 02 54](https://github.com/woocommerce/woocommerce/assets/1314156/4ce04e0a-faaf-4810-b063-978bd513f941)

**Acceptance criteria**

- [ ] We update the help text according to the design: AkVNGImLgSqCObTQ3idVn7-fi-3301_392136
- [ ] The `product settings` bit opens WooCommerce > Settings > Products > Downloadable products in a new tab.

Closes #41465.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a WooCommerce store with this branch and enable the product editor using **WooCommerce > Settings > Advanced > Features**, also enable the `product-virtual-downloadable` feature.
2. Go to **Products > Add New > General**.
3. Go to the `Downloads` section and press the tooltip `?`.
4. Verify that the `product settings` link points to **WooCommerce > Settings > Products > Downloadable products**



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
